README.md

```diff
@@ -34,13 +34,11 @@ android {
 ```groovy
 dependencies {
     ...
-    implementation 'com.azure.android:azure-communication-ui-calling:<latest stable release version>'
+    implementation 'com.azure.android:azure-communication-ui-calling:1.1.0'
```

Comment:
We may want to link Maven in this doc as well.
https://search.maven.org/artifact/com.azure.android/azure-communication-ui-calling

Maybe in further reading or near here.

In case we either forget to update, or a Dev wants more info on releases, frequency, beta's etc.

Reply:
Sure. We added GitHub release page link in our beta release. To main consistency, removed that here for GA with both iOS and Android member's consensus. But yes, we could add it in the further reading or having a Related Link section where git samples & maven link could go.

Reply:
Added the maven link in the readme. :)